documentation/agent/usage.md

```diff
@@ -35,27 +58,47 @@ by sending HTTP requests to the agent.
 
 Example:
 
-    $> fusioninventory-agent --server http://my.server/plugins/fusioninventory/ --daemon
+    $> fusioninventory-agent --server http://my.server --daemon
+
+That's the easiest mode to set up, offering (almost) complete control from a
+centralized point, fully compatible with all available agent features, and the
+most flexible in term of usage. However, that's also the most expensive in term
+of resources usage, and the less secure. And the more important: who controls
```

Comment:
I don't like this sentence: `However, that's also the most expensive in term of resources usage, and the less secure.`
You should better add a dedicated paragraph to discuss this point, maybe under the `Usage caution` section. I agree having a daemon/service means few memory resources are always used, but in half-managed mode & autonomous mode, you'll have also few more CPU & I/O resources used as you need to start the perl interpreter and compile the perl code more often.
Regarding security concern, you're suggesting agent is not a secure tool. But this is not true and the security concern is better a security policy concern and even a configuration problem: if a company wants to limit opened ports, it is just possible to not activate the agent http server.